ui/public/locales/de_DE.json

```diff
@@ -1561,12 +1561,12 @@
 "label.previous": "Vorherige",
 "label.primary": "Primär",
 "label.primary.network": "Hauptnetzwerk",
-"label.primary.storage": "Hauptspeicher",
-"label.primary.storage.allocated": "Zugewiesener Hauptspeicher",
-"label.primary.storage.count": "Hauptspeicher-Pools",
-"label.primary.storage.used": "Genutzter Hauptspeicher",
-"label.primarystoragelimit": "Hauptspeicher-Limits (GiB)",
-"label.primarystoragetotal": "Hauptspeicher",
+"label.primary.storage": "Primärspeicher",
```

Comment:
wieso `Haubt`-Netzwerk und `Primär`-speicher

Reply:
Da Hauptspeicher auch mit Arbeitsspeicher verwechselt werden könnte. Deshalb diese Abgrenzung. @DaanHoogland

Reply:
dank u wel ;)

Reply:
Yeah everything is alright. I just wrote in German because your first comment was in German. But lets keep speaking English. ;)

Reply:
I like any language @j-porsche , even German :grinning: , but you are right, I'll refrain from speaking dutch here.